### PR TITLE
Add string comparison to boolean expressions

### PR DESCRIPTION
**Pull request by roryabraham**
### Details
Hopefully fixed a myriad of GH workflow failures caused by [this PR](https://github.com/Expensify/App/pull/4796)

### Fixed Issues
$ n/a

### Tests
Same as https://github.com/Expensify/App/pull/4796

### QA Steps
None.

### CODE REVIEW

**Review by flodnv — COMMENTED** [review]
Why are they not real booleans? Is that a GH Actions thing or are we holding it wrong?

**Comment by roryabraham** [comment]
> Why are they not real booleans? Is that a GH Actions thing or are we holding it wrong?

I think this is a subtle thing with GH actions where inputs and outputs to jobs and actions are always JSON-encoded. So what was a boolean in one job becomes a string in another

**Review by flodnv — DISMISSED** [review]

File: .github/workflows/finishReleaseCycle.yml [hunk 1]
```diff
@@ -33,15 +33,15 @@ jobs:
...
+        if: ${{ steps.validateActor.outputs.isTeamMember == 'true' }}
         id: checkDeployBlockers
         uses: Expensify/App/.github/actions/checkDeployBlockers@main
         with:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           ISSUE_NUMBER: ${{ github.event.issue.number }}
 
       - name: Reopen and comment on issue
-        if: ${{ steps.validateActor.outputs.isTeamMember && steps.checkDeployBlockers.outputs.HAS_DEPLOY_BLOCKERS == 'true' }}
+        if: ${{ steps.validateActor.outputs.isTeamMember == 'false' && steps.checkDeployBlockers.outputs.HAS_DEPLOY_BLOCKERS == 'true' }}
```
Comment (AndrewGable) [inline .github/workflows/finishReleaseCycle.yml:44]: This changed from `steps.validateActor.outputs.isTeamMember` to `steps.validateActor.outputs.isTeamMember == 'false'`- Was the logic wrong or should it be `true`?

Comment (roryabraham) [reply .github/workflows/finishReleaseCycle.yml]: Good catch, that was a mistake it should have been `true`

**Review by AndrewGable — APPROVED** [review]

**Comment by roryabraham** [comment]
Let's see if this works 🤞

**Comment by roryabraham** [comment]
Seems like it did not work 100%, but has much less severe issues than before: https://github.com/Expensify/App/runs/3465096540?check_suite_focus=true

**Pull request merged**